modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -595,6 +596,21 @@ public Map<String, ModelsMap> postProcessAllModels(Map<String, ModelsMap> objs)
         return processed;
     }
 
+    private boolean modelIsMutatable(CodegenModel model, Set<String> processed) {
```

Comment:
can you please add a 1 liner explaining what this function does? what is classified as mutatable?

Reply:
done